backend/src/test/java/gov/cdc/usds/simplereport/db/model/FeatureFlagTest.java

```diff
@@ -0,0 +1,40 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class FeatureFlagTest {
+  FeatureFlag featureFlag = new FeatureFlag();
```

Comment:
Can you change this variable to be `private`? Otherwise it will default to `protected`, which _shouldn't_ be an issue but better to be cautious